src/document.h

```diff
@@ -89,13 +89,13 @@ struct Document {
 
     #define loopcellsin(par, c) \
         CollectCells(par);      \
-        loopv(_i, itercells) for (Cell *c = itercells[_i]; c; c = nullptr)
+        loopv(_i, itercells) for (Cell *&c = itercells[_i]; c; c = nullptr)

```

Comment:
What is this for? The only thing this would do is allow the loop code to modify `c`, but we don't have any use for that currently.